### PR TITLE
T0000 Implement `classNames` lib styling solution

### PR DESCRIPTION
**Pull request by bmarty92**
- Installed `classNames` lib to improve conditional styling.
- Renamed classNames of some components for consistency.

### CODE REVIEW

File: src/components/SubPage/SubPage.css [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .SubPage {
   margin: calc(var(--s__unit) * 8) 0;
 
-  &.has-image {
+  &__with-image {
```
Comment (grinkus-adapt) [inline src/components/SubPage/SubPage.css:4]: shouldn't this be a modifier?

File: src/pages/bukime-budrus/index.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ const Page = ({ data }) => {
 
       {!!content && (
         <Constraint>
-          <Breadcrumb className="__solo" crumbs={crumbs} />
```
Comment (grinkus-adapt) [inline src/pages/bukime-budrus/index.js:33]: The solo thing is not even needed. Wrap breadcrumbs in `NavigationGroup` to apply the same styling as in other pages.

File: src/components/Breadcrumbs/Breadcrumb.css [hunk 3]
```diff
@@ -26,7 +26,7 @@
     padding: 0;
   }
 
-  &__solo {
+  &--solo {
```
Comment (grinkus-adapt) [inline src/components/Breadcrumbs/Breadcrumb.css:29]: https://github.com/adaptdk/su-ukraina/pull/71/files#r850128834

File: src/components/SubPage/SubPage.css (hunk 1 above)
Comment (bmarty92) [reply src/components/SubPage/SubPage.css:4]: According to the current logic `SubPage` component could have an image which triggers its custom className - `SubPage__with-image`.
If there's image alignment modifier (prop) we're conditionally adding a modifier `SubPage__with-image--left-aligned` or `SubPage__with-image--right-aligned`.

Comment (grinkus-adapt) [reply src/components/SubPage/SubPage.css:4]: Your explanation further proves my point -- that's a modifier, not an element inside a block.

Comment (bmarty92) [reply src/components/SubPage/SubPage.css]: Now I got the point! Fixed this issue.

File: src/components/SubPage/SubPage.css [hunk 4]
```diff
@@ -12,43 +12,43 @@
...
 
-    &.has-image--right {
-      .SubPage__content-image {
-        @media (min-width: 40rem) {
-          margin-left: 20px;
-        }
+  &--image-right-aligned {
+    .SubPage__content-image {
+      @media (min-width: 40rem) {
+        margin-left: 20px;
```
Comment (grinkus-adapt) [inline src/components/SubPage/SubPage.css:42]: DRY.

Simply add the rules that are duplicated in a single block of `&--image-left-aligned, &--image-right-aligned`, and then do the specific rules after.

```
.a, .b { thing1, thing2 }
.a { thing3 }
```
vs
```
.a { thing1, thing2, thing3 }
.b { thing1, thing2 }
```

**Pull request merged**